service/src/main/kotlin/app/cash/backfila/ui/components/DashboardLayout.kt

```diff
@@ -0,0 +1,40 @@
+package app.cash.backfila.ui.components
+
+import kotlinx.html.TagConsumer
+import kotlinx.html.div
+import kotlinx.html.main
+import kotlinx.html.script
+import misk.web.dashboard.HtmlLayout
+
+fun TagConsumer<*>.DashboardLayout(
+  title: String,
+  path: String,
+  block: TagConsumer<*>.() -> Unit = {},
+) {
+  HtmlLayout(
+    appRoot = "/",
+    title = title,
+    // TODO only use play CDN in development, using it always for demo purporses to avoid UI bugs
+//    playCdn = deployment.isLocalDevelopment,
```

Comment:
Do we want to delete this then?

Reply:
I might need them later, will cleanup in follow up.